[bug] word-break properties do not work

**Describe the bug**
There is no way to use `break-normal`, `break-words`, `break-all`, and `break-keep`.

**Expected behavior**
That the project builds successfully.

**Actual behavior**
Build fails with:
```
error TS2353: Object literal may only specify known properties, and 'wordBreak' does not exist in type 'WindTypes'.

27         wordBreak: "break-words",
           ~~~~~~~~~

  node_modules/tailwindest/dist/index.d.ts:198:9
    198         base?: StyleType;
                ~~~~
    The expected type comes from property 'base' which is declared here on type '{ truthy: WindTypes; falsy: WindTypes; base?: WindTypes | undefined; }'
```

**Additional context**
After a quick search, the code to make this work already exists, but is not referenced in the code base. A pull request with a fix will be submitted soon.

- Fedora Linux 40 (KDE Plasma) x86_64
- tailwindest 2.3.0
- tailwindcss 3.4.3
- react 18.3.1
- vite 4.5.3
- typescript 5.4.5

--- a/packages/tailwindest/src/types/tailwind/properties/font/index.ts
+++ b/packages/tailwindest/src/types/tailwind/properties/font/index.ts
@@ -25,6 +25,7 @@ import { TailwindTextUnderlineOffsetType } from "./@text.underline.offset"
 import { TailwindTextWrapType } from "./@text.wrap"
 import { TailwindVerticalAlignType } from "./@vertical.align"
 import { TailwindWhitespaceType } from "./@whitespace"
+import { TailwindWordBreakType } from "./@word.break"
 
 export interface TailwindFontPlug {
     content?: string
@@ -94,4 +95,5 @@ export interface TailwindFont<
         TailwindHyphensType<FontPlug["hyphens"]>,
         TailwindListStylePositionType,
         TailwindVerticalAlignType,
-        TailwindWhitespaceType {}
+        TailwindWhitespaceType,
+        TailwindWordBreakType {}

